core/notifications/src/email_executor/config.rs

```diff
@@ -0,0 +1,8 @@
+use serde::{Deserialize, Serialize};
+
+use super::lettre::LettreConfig;
+
+#[derive(Clone, Default, Debug, Deserialize, Serialize)]
+pub struct EmailExecutorConfig {
+    pub lettre: LettreConfig,
```

Comment:
Don’t want to expose internals like that. Change the smtp library should not be a breaking change